exec/plugin-commands-script-runners/src/run.ts

```diff
@@ -9,6 +9,12 @@ import { type CompletionFunc } from '@pnpm/command'
 import { prepareExecutionEnv } from '@pnpm/plugin-commands-env'
 import { FILTERING, UNIVERSAL_OPTIONS } from '@pnpm/common-cli-options-help'
 import { type Config, types as allTypes } from '@pnpm/config'
+import {
+  type CheckLockfilesUpToDateOptions,
+  DISABLE_DEPS_CHECK_ENV,
+  checkLockfilesUpToDate,
+  shouldRunCheck,
```

Comment:
this function and the env variable should not be part of the deps.status package.

Reply:
https://github.com/pnpm/pnpm/pull/8645/commits/69528f0edf974fa9cbe6be01e3706590c58205ce